lib/pause_2017/templates/user/mfa/edit.html.ep

```diff
@@ -0,0 +1,59 @@
+% layout 'layout';
+% my $pause = stash(".pause") || {};
+% my $cpan_alias = lc($pause->{HiddenUser}{userid}) . '@cpan.org';
```

Comment:
I think `$cpan_alias` is unused in this file

Reply:
Removed $cpan_alias with https://github.com/andk/pause/pull/455/commits/ca5ea284250293120610be611bbf0302292ff933